hazelcast/src/main/java/com/hazelcast/instance/GroupProperties.java

```diff
@@ -57,11 +57,32 @@
      * <p/>
      * The performance monitor logs all metrics into the log file.
      */
-    public static final String PROP_PERFORMANCE_MONITORING_ENABLED = "hazelcast.performance.monitoring.enabled";
+    public static final String PROP_PERFORMANCE_MONITOR_ENABLED = "hazelcast.performance.monitoring.enabled";
+
+    /**
+     * The delay in seconds between monitor of the performance.
+     */
+    public static final String PROP_PERFORMANCE_MONITOR_DELAY_SECONDS
+            = "hazelcast.performance.monitor.delay.seconds";
+
+    /**
+     * The PerformanceMonitor uses a rolling file approach to prevent eating too much disk space.
+     *
+     * This property sets the maximum size in MB for a single file.
+     *
+     * Every HazelcastInstance will get its own history of log files.
+     */
+    public static final String PROP_PERFORMANCE_MONITOR_MAX_ROLLED_FILE_SIZE
+            = "hazelcast.performance.monitor.max.rolled.file.size";
+
     /**
-     * The delay in seconds between monitoring of the performance.
+     * The PerformanceMonitor uses a rolling file approach to prevent eating too much disk space.
+     *
+     * This property sets the maximum number of rolling files to keep on disk.
      */
-    public static final String PROP_PERFORMANCE_MONITORING_DELAY_SECONDS = "hazelcast.performance.monitoring.delay.seconds";
+    public static final String PROP_PERFORMANCE_MONITOR_MAX_ROLLED_FILE_COUNT
```

Comment:
can document the default value + describe a way how to disable it? (if there is any)